library/fiber/raise.rb

```diff
@@ -0,0 +1,91 @@
+require_relative '../../spec_helper'
+
+require 'fiber'
+
+ruby_version_is "2.7" do
+  describe "Fiber#raise" do
+
+    it 'raise RuntimeError by default' do
+      fiber = Fiber.new { Fiber.yield }
+      fiber.resume
+      -> { fiber.raise }.should raise_error(RuntimeError)
+    end
+
+    it "raise FiberError if Fiber is not born" do
+      fiber = Fiber.new { true }
+      -> { fiber.raise }.should raise_error(FiberError, "cannot raise exception on unborn fiber")
+    end
+
+    it "raise FiberError if Fiber is dead" do
+      fiber = Fiber.new { true }
+      fiber.resume
+      -> { fiber.raise }.should raise_error(FiberError, "dead fiber called")
+    end
+
+    class TestError < StandardError ; end
```

Comment:
This will define a new global constant, it should instead be nested under `module FiberSpecs`, or `Class.new(StandardError)` be used instead.